### ISSUE TITLE
Multiple Website issue

### ISSUE THREAD

**Issue by armindoPR** [organicinternet/magento-configurable-simple#18]
Hi,

first of all this module is great, 
second i want to know if is possible to setup this module with multiple websites (System->Manage Stores-> Create Website). The problem is i want this module enable for one of the websites and disable for the rest.

Thanks !!!

**Comment by toasty** [comment]
Hiya, 

At the moment that's not possible, sorry. SCP changes core behaviour that's lower down than the Magento 'website' level.

It's probably doable in theory, but it certainly wouldn't be easy! You'd probably have to go through a large part of the code and add lots of checks to see whether the particular website in question had SCP 'turned on' or not.

Matt

**Comment by armindoPR** [comment]
It will be nice if in a future release we have that option to enable/disable the module for specific website/store.

Thanks !!!

**Comment by toasty** [comment]
Agreed. On my todo list I've had enable/disable SCP per-product for ages, but it's one of those things that's a lot of work to implement and test, and I think relatively few people need, so it always seems to stay low down on the list of priorities.  I'll add per-store/per-website to the todo too. Can't promise when it'll happen though.

**Comment by rwanninkhof** [comment]
Maybe we can sponsor you to add this multi-store/multi-website feature?
